asv_bench/benchmarks/benchmarks.py

```diff
@@ -1184,4 +1184,96 @@ def time_series_indices(self, shape):
         self.series.groupby(self.series).indices
 
 
+class TimeRepr:
+
+    params = [get_benchmark_shapes("TimeRepr")]
+    param_names = ["shape"]
+
+    def setup(self, shape):
+        self.df = IMPL.DataFrame(np.random.randn(*shape))
+        execute(self.df)
+
+    # returns a  string thus not calling execute
+    def time_repr(self, shape):
+        repr(self.df)
+
+
+class TimeMaskBool:
+
+    params = [get_benchmark_shapes("TimeMaskBool")]
+    param_names = ["shape"]
+
+    def setup(self, shape):
+        df = IMPL.DataFrame(np.random.randn(*shape))
+        df = df.where(df > 0)
+        self.bools = df > 0
+        self.mask = IMPL.isnull(df)
+        execute(self.bools), execute(self.mask)
+
+    def time_frame_mask_bools(self, shape):
```

Comment:
Let's drop `time_frame_mask_bools` case.

Reply:
dropped the previous time_dropna case and renamed time_dropna_axis_mixed_dtypes to time_dropna